cookbook/validation/validation-group-service-resolver.rst

```diff
@@ -0,0 +1,55 @@
+How to dynamically configure validation groups
```

Comment:
according to our title case standards, this should be "How to Dynamically Configure Validation Groups"